doc/changes/latest.rst

```diff
@@ -26,6 +26,7 @@ Fixes
 - Fix usage of ``scipy.stats.gamma.pdf`` in ``_gamma_difference_hrf`` function under ``nilearn/glm/first_level/hemodynamic_models.py``, which resulted in slight distortion of HRF (:gh:`3297` by `Kun CHEN`_).
 - Fix bug introduced due to a fix in the pre-release version of scipy (``1.9.0rc1``) which now enforces that elements of a band-pass filter must meet condition ``Wn[0] < Wn[1]``.
   Now if band-pass elements are equal :func:`~nilearn.signal.butterworth` returns an unfiltered signal with a warning (:gh:`3293` by `Yasmin Mzayek`_).
+- Fix bug in behavior of ``alpha`` in ``nilearn.plotting.plot_connectome`` (:gh:`3306` by `Koen Helwegen`_).
```

Comment:
```suggestion
- Fix bug in behavior of ``alpha`` in :func:`~plotting.plot_connectome` (:gh:`3306` by `Koen Helwegen`_).
```
The description can also be a bit more explicit, such as 

The parameter \`\`alpha\`\` is now correctly passed to :func:\`~plotting.plot_glass_brain\` in :func:\`~plotting.plot_connectome\`

Reply:
Thanks, updated in [e37b9ce](https://github.com/nilearn/nilearn/pull/3306/commits/e37b9cea58ec59ce3b7a36f29d6ffe56b37afd2f)